common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,31 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  /**
+   * When memory usage of Netty is above this water mark, it's regarded as memory shortage.
```

Comment:
maybe we can create a JIRA and send it to dev-list to gather feedbacks. Do you mind to do this if you have time?